html_app/ui/MicroscopyView.js

```diff
@@ -385,16 +383,29 @@ scb.ui.static.MicroscopyView.scb_f_microscopy_select_slide_type = function (elem
 				return;
 				
 		}
-       if(_.size(parsed.assignment.template.micro_kinds[slide_type].conditions) == 1 || 
-       	  _.size(parsed.assignment.template.slide_parser[parsed.experiment.cell_treatment_list.get(cell_treatment_id).treatment_list.first.collection_id][slide_type])==1 || 
+        /*
+        If there is one condition
+         */
+        var cond_num_collection_id= 0, cond_num_default=0;
+        if (parsed.assignment.template.slide_parser[parsed.experiment.cell_treatment_list.get(cell_treatment_id).treatment_list.first.collection_id]){
+            cond_num_collection_id=_.size(parsed.assignment.template.slide_parser[parsed.experiment.cell_treatment_list.get(cell_treatment_id).treatment_list.first.collection_id][slide_type]);
```

Comment:
@bdero do you have JavaScript style advice on dealing with these super-long lines?